### ISSUE TITLE
UTF-8 encoded Email subjects like "=?UTF-8?Q? Personal=C3=A4nderung?=" not decoded

### ISSUE THREAD

**Issue by SummerStorm** [zammad/zammad#2456]
### Infos:

* Used Zammad version: 2.8
* Installation method (source, package, ..): any
* Operating system: any
* Database + version: any
* Elasticsearch version: any
* Browser + version: any

### Expected behavior:

* the parsed subject should be `Personaländerung`

### Actual behavior:

* the parsed subject is `=?UTF-8?Q?Personal=C3=A4nderung?=`

### Steps to reproduce the behavior:

* Parse an Email with a subject header that's an UTF-8 quoted printable string like `=?UTF-8?Q?Personal=C3=A4nderung?=`

Yes I'm sure this is a bug and no feature request or a general question.

This is related to #1045, which is about parsing a us-ascii quoted printable string subject. 

EDIT: The original report contained an incorrect triggering string: `=?UTF-8?Q?Personal=C3=A4nderung?=`. Thanks to @martinvonwittich for pointing out the mistake. The correct   triggering string is `=?UTF-8?Q? Personal=C3=A4nderung?=`, with an added significant space character.

**Comment by martinvonwittich** [comment]
For some reason I'm unable to reproduce this. My Zammad 2.8.x instance seems to handle both these cases fine:

- `Subject: =?utf-8?Q?Personal=C3=A4nderung?=`
- `Subject: =?UTF-8?Q?Personal=C3=A4nderung?=`

The complete mail source:

```
Return-Path: <redacted@redacted>
Received: from redacted (localhost [127.0.0.1])
	 by redacted (Cyrus 2.5.10-Debian-2.5.10-3+1~iserv10809+10869-1) with LMTPA;
	 Wed, 30 Jan 2019 00:02:32 +0100
X-Sieve: CMU Sieve 2.4
Received: from localhost ([127.0.0.1] helo=redacted)
	by redacted with esmtpa (Exim 4.89)
	(envelope-from <redacted@redacted>)
	id 1gocOm-0007PR-Cy
	for redacted@redacted; Wed, 30 Jan 2019 00:02:32 +0100
Message-ID: <9a21f01195853f35cb86c66c64bb2d5c@redacted>
Date: Wed, 30 Jan 2019 00:02:32 +0100
Subject: =?UTF-8?Q?Personal=C3=A4nderung?=
From: Martin von Wittich <redacted@redacted>
To: zammad <redacted@redacted>
MIME-Version: 1.0
Content-Type: text/plain; charset=utf-8
Content-Transfer-Encoding: quoted-printable
X-Mailer: IServ v3

test
```

**Comment by SummerStorm** [comment]
Thanks for double checking, @martinvonwittich! You are correct, I made a mistake in the bug report. The offending string is `=?UTF-8?Q? Personal=C3=A4nderung?=`, with a space in front of the string. The original subject was a sentence containing the word "Personaländerung",  with a space in front of the word itself. I accidentally stripped the space in front off, thinking that it's purely the umlaut causing the problem. But alas, the extra space is actually critical in activating the bug.

I've edited the issue title and my OP, along with an edit note.

**Comment by SummerStorm** [comment]
The previously incorrect test cases have been fixed via 0b472fa.

**Comment by mohiwalla** [comment]
Hello everyone I think I am quite late here, nevertheless I think new devs will have answer for there problem. You can actually user `imap_utf8()` to decode these kind of strings. I have also tried many other functions but none of those work at all. Enjoy coding :D

-mohiwalla
